### PR TITLE
Release v0.1.0

### PR DESCRIPTION
**Pull request by zmbc**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        run: |
+          python --version
+          python -m pip install --upgrade pip
+          pip install setuptools wheel build
+      - name: Build
+        run: python -m build
+      - name: Publish
+        uses: pypa/gh-action-pypi-publish@release/v1
+      - name: Send mail
+        # Notify when cron job fails
```
Comment (patricktnast) [inline .github/workflows/deploy.yml]: nit--I know we do this all over our framework deploy yamls too but we probably should change the (presumably copy pasted) reference to a "cron job"

**Review by patricktnast — COMMENTED** [review]
I'm sure there's a good reason, but I'm not very familiar with this--why use a new repo with a single submodule instead of releasing `person_linkage_case_study` directly?

**Comment by zmbc** [comment]
@patricktnast Oh sorry, you were probably on vacation when I discussed this with Rajan. The reason for the new repo is that I want the "primary" release to be the one on GitHub -- most people will use that one -- and I want everything to be at the top level there. This PyPI release is secondary and requires stuff to be in subdirectories in order to make them Python packages.

**Comment by zmbc** [comment]
It also allows me to create a second README for how to use the (weird) PyPI version without confusing people over at the main repo.

**Comment by patricktnast** [comment]
I don't think I understand the distinction, but if you talked it over with Rajan I don't want to block you from wrapping this up. I'll approve but maybe you can explain this to me on zoom at some point?

**Review by patricktnast — APPROVED** [review]

**Pull request merged**